doozer/doozerlib/metadata.py

```diff
@@ -401,11 +402,121 @@ async def get_latest_build(self, **kwargs):
             return await self.get_latest_brew_build_async(**kwargs)
 
         elif self.runtime.build_system == 'konflux':
-            raise NotImplementedError
+            return await self.get_latest_konflux_build(**kwargs)
```

Comment:
Considering this option is only to switch implementations for finding builds and ideally won't cause any user-facing differences, this might be better to be controlled by an env var. With that env var, we can start to having certain jobs to use the konflux implementation, then gradually make that default and remove the brew implementation without bothering removing this option.

But this is only an option suggestion and I can live with this if you have strong opinion.

Reply:
I don't either, also an env var sounds good to me. Would adding a `envar=BUILD_SYSTEM` [here](https://github.com/openshift-eng/art-tools/blob/main/doozer/doozerlib/cli/__init__.py#L107) work?

Reply:
Seems Doozer defines a lot of global env vars here: https://github.com/openshift-eng/art-tools/blob/27708d09b004e12ab590f95ca5e42271cadb7475/doozer/doozerlib/cli/cli_opts.py#L28

Reply:
Done with 5730f753935092b0ec307f3b5bb2c4a2d2af4eb2